configs/corpus_config.py

```diff
@@ -0,0 +1,40 @@
+import sys
+sys.path.insert(0,r'./')
+from .config import Config
+from typing import List, Dict
+from dataclasses import dataclass, asdict, fields
+
+
+@dataclass
+class CorpusConfig(Config):
+    """
+    A single training/test example for base config.
+    """
+    system_prompt: str
```

Comment:
I don't really think a Corpus dataset will need a system_prompt attr, so you can remove this